### PR TITLE
Div

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/core/regular/div.hpp [hunk 1]
```diff
@@ -8,12 +8,33 @@
...
+    EVE_FORCEINLINE constexpr
+    kumi::apply_traits_t<eve::common_value,Tup>
+    operator()(Tup const& t) const noexcept requires(kumi::size_v<Tup> >= 2) { return EVE_DISPATCH_CALL(t); }
+
+    EVE_CALLABLE_OBJECT(div_t, div_);
+  };
+
 //================================================================================================
-//! @addtogroup core_arithmetic
+//! @divtogroup core_arithmetic
```
Comment (jfalcou) [inline include/eve/module/core/regular/div.hpp:37]: @addtogroup

**Review by jfalcou — CHANGES_REQUESTED** [review]
- Did you forget to remove the div and rem from operators.hpp ?
- I thought we wanted fmod to be rem for float and rem not supporting float anymore ?

File: include/eve/module/core/regular/rem.hpp [hunk 2]
```diff
@@ -96,11 +112,11 @@ namespace eve
...
-EVE_MAKE_CALLABLE(rem_, rem);
+  inline constexpr auto rem = functor<rem_t>;
 }
 
 #include <eve/module/core/regular/impl/rem.hpp>
 
-#if defined(EVE_INCLUDE_ARM_HEADER)
-#  include <eve/module/core/regular/impl/simd/arm/neon/rem.hpp>
-#endif
+// #if defined(EVE_INCLUDE_ARM_HEADER)
```
Comment (jfalcou) [inline include/eve/module/core/regular/rem.hpp:120]: Remove or uncomment ?

File: include/eve/module/core/regular/fmod.hpp [hunk 3]
```diff
@@ -7,18 +7,77 @@
...
+//!   #include <eve/module/core.hpp>
+//!   @endcode
+//!
+//!   @groupheader{Callable Signatures}
+//!
+//!   @code
+//!   namespace eve
+//!   {
+//!      template< eve::value T,  eve::value U>
+//!      eve::common_value_t<T, U> rem(T x, U y) noexcept;
```
Comment (jfalcou) [inline include/eve/module/core/regular/fmod.hpp:48]: rem is fmod here

File: include/eve/module/core/regular/div.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/core/regular/div.hpp:37]: lol done

File: include/eve/module/core/regular/rem.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/core/regular/rem.hpp:120]: removed

File: include/eve/module/core/regular/fmod.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/module/core/regular/fmod.hpp:48]: done and put floating_value as doc says

File: include/eve/module/core/regular/rem.hpp [hunk 4]
```diff
@@ -96,11 +112,7 @@ namespace eve
...
 //! @}
 //================================================================================================
-EVE_MAKE_CALLABLE(rem_, rem);
+  inline constexpr auto rem = functor<rem_t>;
 }
 
 #include <eve/module/core/regular/impl/rem.hpp>
-
-#if defined(EVE_INCLUDE_ARM_HEADER)
-#  include <eve/module/core/regular/impl/simd/arm/neon/rem.hpp>
```
Comment (jfalcou) [inline include/eve/module/core/regular/rem.hpp:105]: maybe we can remove the file then

**Pull request merged**